sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2472,27 +2472,49 @@ object Radar {
                     return
                 }
 
-                sendEvent(customType, metadata, location, events, user, callback)
+                logConversion(customType, data, location, events, user, callback)
             }
         })
     }
 
     /**
-     * Sends a custom event.
+     * Logs a conversion with a manually provided location.
      *
      * @see [](https://radar.com/documentation/api#send-a-custom-event)
      *
-     * @param[customType] The name of the event.
-     * @param[metadata] The metadata associated with the event.
+     * @param[customType] The name of the conversion.
+     * @param[revenue] The revenue generated by the conversion.
+     * @param[data] The metadata associated with the conversion.
+     * @param[callback] A callback.
+     */
+    @JvmStatic
+    fun logConversion(
+        customType: String,
+        revenue: Float,
+        data: JSONObject?,
+        callback: RadarLogConversionCallback
+    ) {
+        data?.put("revenue", revenue);
```

Comment:
does `data` need to be initialized as an empty object / dict if it's not present